examples/echo_fastapi_based_bot.py

```diff
@@ -0,0 +1,45 @@
+from aiogram import Bot, Dispatcher, types
+from fastapi import FastAPI
+import logging
+import asyncio
+
+API_TOKEN = "<TOKEN>"
+
+logging.basicConfig(level=logging.DEBUG)
+
+class AsyncioBot:
+
+    # Initialize bot and dispatcher
+    def __init__(self) -> None:
+        self.bot = Bot(token=API_TOKEN)
+        self.dp = Dispatcher(self.bot)
+
+        self.dp.register_message_handler(self.on_message)
+
+    async def on_message(self, message: types.Message):
+        print(message.get_command())
+        await message.answer(f"Asnwer: {message.text}")
+
+    async def run(self):
+        # await self.dp.start_polling() # will be blocking main :on_start: FastAPI event
+        asyncio.create_task(self.dp.start_polling())
```

Comment:
FastAPI with polling :/
IMO, this doesn't make any sense since you already have a web server running.